src/convolution.cpp

```diff
@@ -779,21 +779,21 @@ std::ostream& operator<<(std::ostream& stream, const ConvolutionDescriptor& c)
     return stream;
 }
 
+/// \todo Error handling
 void ConvolutionAttribute::Set(miopenConvolutionAttrib_t attr, int value) { store[attr] = value; }
 
 int ConvolutionAttribute::Get(miopenConvolutionAttrib_t attr) const
 {
     if(attr == MIOPEN_CONVOLUTION_ATTRIB_FP16_ALT_IMPL)
     {
-        auto p = miopen::GetStringEnv(MIOPEN_DEBUG_FP16_ALT_IMP{});
-        if(p != nullptr)
-        {
+        if(nullptr != miopen::GetStringEnv(MIOPEN_DEBUG_FP16_ALT_IMP{}))
```

Comment:
@carlushuang Basically we shouldn't override the value returned by "Get", but let's keep this (for speed).